language/samples/v1/samplegen_map_field_access.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# DO NOT EDIT! This is a generated sample ("Request",  "samplegen_map_field_access")
+
+# To install the latest published package dependency, execute the following:
+#   pip install google-cloud-language
+
+# sample-metadata
+#   title: This sample reads and loops over a map field in the response
+#   description: This sample reads and loops over a map field in the response
+#   usage: python3 samples/v1/samplegen_map_field_access.py
+
+# [START samplegen_map_field_access]
+from google.cloud import language_v1
+from google.cloud.language_v1 import enums
+
+
+def sample_analyze_entities():
+    """This sample reads and loops over a map field in the response"""
+
+    client = language_v1.LanguageServiceClient()
+
+    type_ = enums.Document.Type.PLAIN_TEXT
+    language = "en"
+
+    # The text content to analyze
+    content = "Googleplex is at 1600 Amphitheatre Parkway, Mountain View, CA."
+    document = {"type": type_, "language": language, "content": content}
+
+    response = client.analyze_entities(document)
+    for entity in response.entities:
+        # Access value by key:
+        print(u"URL: {}".format(entity.metadata["wikipedia_url"]))
```

Comment:
w00t! Glad to hear this, I'd love to use `f""` formatting 💕

I believe we're close enough now to the Python 2 deprecation to switch this over!

Same with removing the `u""` ~ some developers had gotten bitten by encodings when using Python samples from the website, so we added `u""` because it was a very lightweight way to keep developers from hurting themselves. But, in the context of deprecating Python 2, I'd love to see this fully moved over to `f"Hello {var}"` without the `u""`

+💯

Reply:
The target Python 3 versions supported are >= 3.5, which precludes the use of fstrings as they were introduced in 3.6

Reply:
☹️

How about we do this?

1. Remove `u""`
1. Don't use `f""` (**_yet_**)